docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+
+When uploading mods to Nexus Mods, there are a few things to keep in mind. First, you'll want to make sure that what you're uploading is unique and has not already been released by another modder. For example, since azirus3 has already uploaded an NPC swap pack that includes Silvio Caruso, we probably wouldn't want to upload our swap as a stand-alone mod. Also, you'll want to consider if what you're releasing will need its own stand-alone page and will appeal to as many users as possible. For example, if we were to create a number of small suit modifications and release them all as separate mods, it could lead to a lot of clutter (think ReShade presets). But if we were to collect all those modifications and release them as a single mod, not only would it make things less cluttered, but it would also be easier for the end user as everything would be available in one place.
+
+Of course, you are free to release whatever you'd like as long as it doesn't break any rules or include other people's work without their permission. These are only guidelines for those new to the modding scene.
+
+# TL;DR
+1. Import the Runtime folder into RPKG Tool.
+2. Find the outfit_\*\*\*\*\*\*" TEMP file for the outfit/NPC you're looking for.
+3. Extract both the TEMP's QuickEntity JSON and the recursive hash depends to the chunk0 folder in your Simple Mod Framework mod.
+4. Find the TEMP value for the suit you want to replace.
+5. Open the outfit's \*\*\*\*\*\*.entity.json and replace its tempHash value with the value you found in step 4.
+6. Open the \*\*\*\*\*\*.TEMP folder within the ALLDEPENDS folder. You will only need the depends from the chunk the suit is in (e.g. if the suit is in chunk1, you will only need the files from chunk1 and its patches). Move everything in each patchX folder to the outfit's base chunkX folder (e.g. chunk1patch1 to chunk1, then chunk1patch2 to chunk1, overwriting anything in the base chunk if necessary).
+7. Move all the depends from the outfit's base chunkX folder to the chunk0 folder in your SMF mod (the same place your ******.entity.json is located).
+8. Delete whatever's remaining in the ALLDEPENDS folder.
+9. Make sure the folder structure for your Simple Mod Framework mod is correct, with the JSON/Depends in a chunk0 folder, and make sure there are no errors in your mod's manifest.json.
+10. Enable the mod in Simple Mod Framework and then click "Apply Enabled Mods".
+
+# Credits
+`Metaphoria` - Tutorial author.
+
+`Oakheart` - Editing and conversion to markdown, as well as the V2 chunk flowchart.
+
+`invalid` - Original chunk flowchart, of which V2 was based.
+
+`Notex` & `REDACTED` - RPKG Tool.

```

Comment:
This is a gripe I've had for a while. It is not just Notex and REDACTED anymore. It is a plethora of contributors now, REDACTED did the bulk of the work and Notex hosted it and collaborated with them.

Reply:
RPKG Contributors sounds more realistic in this sense.

Reply:
I've changed it to: `Notex`, `REDACTED`, [and all RPKG contributors](https://github.com/glacier-modding/RPKG-Tool/graphs/contributors) - RPKG Tool. 

I did initially list out all the GitHub contributors, but I think linking to the contributors on GitHub is probably the best solution for now.